src/views/landing.njk

```diff
@@ -20,56 +20,56 @@
 {% endblock %}
 
 {% block content%}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
-      <p class="govuk-body-l">Use this service to appeal a penalty issued to a company for not filing its annual accounts on time.</p>
+  <div class='govuk-grid-row'>
```

Comment:
It is more common to have double quotes for HTML code. Rest such as JS or nunjucks could have single quote.